### ISSUE TITLE
Markdown parse error with newline followed by an image

### ISSUE THREAD

**Issue by marcojakob** [decaporg/decap-cms#2421]
**Describe the bug**

NetlifyCMS crashes by trying to open the following markdown file:

```
+++
# Some toml metadata
+++

![](myimage.png)
```

The parse error is caused by the **newline** followed by an **image** (if I remove the newline or the image it works).

The error message also contained a link to report it to `remarkjs`. So, I did that but it is hard for me to tell if it is a Netlify or a Remarkjs problem. 

Here is the issue: https://github.com/remarkjs/remark/issues/413

**Applicable Versions:**

 - Netlify CMS version: 2.9.5
 - Git provider: GitHub
 - OS: Windows 10
 - Browser version: Chrome

**Comment by erquhart** [comment]
Should be resolved by https://github.com/netlify/netlify-cms/issues/2402, close to completion.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by cunneen** [comment]
Downgrading to netlify CMS 2.9.1 (as per #2440) fixed this for me, so the problem seems to have been introduced since then.

**Comment by bblanchon** [comment]
Downgrading to 2.9.1 fixed this for me too.
I'm very surprised to see this marked as "wontfix"... is there really no plan to fix this bug?

**Comment by erquhart** [comment]
Test after #2828 is merged.
